server/mock/mock_types_test.go

```diff
@@ -205,9 +205,10 @@ func TestHexToSignature(t *testing.T) {
 
 	publicKey := hexutil.Encode(bls.PublicKeyToBytes(blsPublicKey))
 
-	message := &builderApiCapella.BuilderBid{
-		Header: &capella.ExecutionPayloadHeader{
-			BlockHash: HexToHash("0xe28385e7bd68df656cd0042b74b69c3104b5356ed1f20eb69f1f925df47a3ab7"),
+	message := &builderApiDeneb.BuilderBid{
+		Header: &deneb.ExecutionPayloadHeader{
+			BlockHash:     HexToHash("0xe28385e7bd68df656cd0042b74b69c3104b5356ed1f20eb69f1f925df47a3ab7"),
+			BaseFeePerGas: uint256.NewInt(0),
```

Comment:
Why did you add BaseFeePerGas here?

Reply:
`BaseFeePerGas` no longer can be nil in the new `deneb.ExecutionPayloadHeader` type